src/rgw/driver/rados/rgw_notify.cc

```diff
@@ -313,14 +354,17 @@ class Manager : public DoutPrefixProvider {
           // bail out on first error
           break;
         }
-        // TODO pass entry pointer instead of by-value
-        spawn::spawn(yield, [this, &queue_name, entry_idx, total_entries, &end_marker, &remove_entries, &entries_to_remove, &has_error, &waiter, entry](yield_context yield) {
+        spawn::spawn(yield, [this, &queue_name, entry_idx, total_entries, &end_marker, &remove_entries, &entries_to_remove, &has_error, &waiter, &entry](yield_context yield) {
```

Comment:
we can probably pass the persistency tracker of the current queue, instead of looking it up multiple times inside the lambda

Reply:
see lines 359 and 367